### DISCUSSION TITLE
Correct version tag for Bandwidth channel?

### DISCUSSION THREAD

**Discussion by alviriseup** [nyaruka/rapidpro#4570] [Q&A] [answered]
Hi team,

It is great to see that the Bandwidth channel has been added, and we are excited to work with it. Also, looking forward to when it comes out of beta user group and into the main channel roster! 

My question is that, there are multiple version tags specifically mentioning the channel type, and I am not sure which one to use.
![Screenshot from 2023-05-11 15-57-31](https://github.com/nyaruka/rapidpro/assets/95750048/6276c5b3-baf3-43ec-88f9-ebd276b7539c)
The CHANGELOG mentions that support for the Bandwidth channel type has been added since v8.1.6. 

Hence, I am a bit confused as to which one to use. We are looking to upgrade from [v7.4.2 (AGPL version)](https://github.com/rapidpro/rapidpro) for the channel.

So, I was wondering if there is a stable release that has the Bandwidth channel integrated (even if only for beta)? In the case we have to upgrade to 8.1.x, what should be the sequence ? e.g. 7.4.2 --> 8.0.1 --> Main Branch off the top?

**Comment by rowanseymour** [comment, marked as answer]
The last stable release on this fork was 8.0.1, and 8.1.x is our current development branch. So if you're setting up a development environment then you should pull the latest 8.1.x of all components. We would not recommend running that as a production instance. If you do switch to 8.0.1 on a production instance be sure to review the Additional Use Grant clause of the [licence](https://github.com/nyaruka/rapidpro/blob/main/LICENSE).

The next stable release on the Nyaruka fork will be 8.2.x sometime in June, at which point 8.0.1 will be upmerged into  the AGPL version.
